crates/mpc-protocols/src/mpc_events.rs

```diff
@@ -1,13 +1,35 @@
+use move_core_types::ident_str;
+use move_core_types::identifier::IdentStr;
+use move_core_types::language_storage::StructTag;
```

Comment:
Consider optimizing imports (putting them in one line)?

Reply:
These imports are from different modules .

Reply:
I don't see how them being from different modules is relevant:

```suggestion
use move_core_types::{ident_str, identifier::IdentStr, language_storage::StructTag};
```